docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -1946,6 +1946,30 @@ alertmanager_client:
   # CLI flag: -ruler.alertmanager-client.basic-auth-password
   [basic_auth_password: <string> | default = ""]
 
+  oauth2:
+    # OAuth2 client ID. Enables the use of OAuth2 for authenticating with
+    # Alertmanager.
+    # CLI flag: -ruler.alertmanager-client.oauth.client_id
+    [client_id: <string> | default = ""]
+
+    # OAuth2 client secret.
+    # CLI flag: -ruler.alertmanager-client.oauth.client_secret
+    [client_secret: <string> | default = ""]
+
+    # Endpoint used to fetch access token from.
+    # CLI flag: -ruler.alertmanager-client.oauth.token_url
+    [token_url: <string> | default = ""]
+
+    # Optional scopes to include with the token request.
+    # CLI flag: -ruler.alertmanager-client.oauth.scopes
+    [scopes: <list of strings> | default = []]
+
+  # (advanced) Optional HTTP, HTTPS via CONNECT, or SOCKS5 proxy URL to route
+  # requests through. Applies to all requests, including infra like oauth token
```

Comment:
`infra` = `infrastructure`?

Reply:
Good catch, that wasn't the greatest wording.
How about: `Applies to all requests, including auxiliary traffic such as OAuth token requests.`
The intent meaning: "this applies all requests related to the client, not just the obvious Alertmanager ones"

Reply:
That sounds great! Just add another comma after traffic:

`Applies to all requests, including auxiliary traffic, such as OAuth token requests.`